R/nodeCentricSteinerTreeProblem-class.R

```diff
@@ -83,7 +84,11 @@ nodeCentricSteinerTreeProblem <- R6Class("nodeCentricSteinerTreeProblem",
       
       if(length(decompose(private$searchGraph)) != 1) stop("Search network must only have a single connected component.")
       
-      # nodeDT with node indicies
+                                             # check for nodeScore: all -1 if absent, validate if present
+                                             if(!"nodeScore" %in% vertex_attr_names( private$searchGraph)){ V(private$searchGraph)$nodeScore <- -1 }
+                                             
```

Comment:
I've added this to make sure the nodeScore are added to searchGraph **even when** presolveGraph is set to FALSE. This was trigering an false warning and stop the suboptimal collection after just 1 iteration.